packages/permission-log-controller/src/permissions.ts

```diff
@@ -0,0 +1,399 @@
+import { type Json, JsonRpcRequestStruct } from '@metamask/utils';
+import deepFreeze from 'deep-freeze-strict';
+
+export const CaveatTypes = Object.freeze({
+  restrictReturnedAccounts: 'restrictReturnedAccounts' as const,
+});
+
+export const RestrictedMethods = Object.freeze({
+  eth_accounts: 'eth_accounts',
+} as const);
+
+/**
```

Comment:
Ah, it looks like you mixed together `shared/constants/permissions.js` and `test/mocks/permissions.js`. 

Typically for other packages we've created a `tests` directory where we can put data and helpers that are used in tests (for instance, see [`network-controller`](https://github.com/MetaMask/core/tree/8d5cd478c5cbdc699a9061643c5137054708506c/packages/network-controller/tests)). What do you think about creating that directory and then moving everything starting from this line onward to a new file there?

Reply:
I have now two files src/constants and tests/helper.ts let me know if that is good @mcmire